### PR TITLE
Auto focus feed incremental daily new

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24377
oldpr: https://github.com/demisto/content/pull/7359

## Description
Added new feed for autofocus daily and updated error while configuring the old AutoFocusFeed on daily.

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by orhovy** [comment]
@kirbles19  Hi, This integration is one command and very similar to AutoFocus feed 
https://github.com/demisto/content/tree/master/Packs/FeedAutofocus/Integrations/FeedAutofocus

File: Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py [hunk 1]
```diff
@@ -422,34 +422,30 @@ def build_iterator(self, limit=None, offset=None):
...
     exception_list = []  # type:List
     if 'Daily Threat Feed' in indicator_feeds:
-        client.indicator_feeds = ['Daily Threat Feed']
-        try:
-            client.build_iterator(1, 0)
-        except Exception:
-            exception_list.append("Could not fetch Daily Threat Feed\n"
-                                  "\nCheck your API key and your connection to AutoFocus.")
-
+        return_error("Daily Feed is no longer supported by this feed,"
```
Comment (bakatzir) [inline Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py:440]: please use raise

File: Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.py [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+            return FeedIndicatorType.File
+
+        # in AutoFocus, URLs include a path while domains do not - so '/' is a good sign for us to catch URLs.
+        elif '/' in indicator:
+            return FeedIndicatorType.URL
+
+        else:
+            return FeedIndicatorType.Domain
+
+    def create_indicators_from_response(self, response, feed_tags):
```
Comment (bakatzir) [inline Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.py:104]: please use type assertion in args and in expected return type

File: Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+    """
+    try:
+        client.build_iterator(argToList(demisto.params().get('feedTags')), 1, 0)
+    except Exception:
+        raise Exception("Could not fetch Daily Threat Feed\n"
+                        "\nCheck your API key and your connection to AutoFocus.")
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client, args: dict, feed_tags):
```
Comment (bakatzir) [inline Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.py:171]: please use type assertion in args and in expected return type

File: Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.py [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+
+    if args.get('limit'):
+        human_readable = human_readable + f"\nTo bring the next batch of indicators run:\n!autofocus-daily-get-indicators " \
+                                          f"limit={args.get('limit')} " \
+                                          f"offset={int(str(args.get('limit'))) + int(str(args.get('offset')))}"
+
+    return human_readable, {}, indicators
+
+
+def fetch_indicators_command(client: Client, feed_tags: List, limit=None, offset=None):
```
Comment (bakatzir) [inline Packs/FeedAutofocus/Integrations/FeedAutofocusDaily/FeedAutofocusDaily.py:206]:
```suggestion
def fetch_indicators_command(client: Client, feed_tags: List, limit=None, offset=None) -> List:
```

File: Packs/FeedAutofocus/pack_metadata.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
     "name": "Autofocus Feed",
     "description": "Indicators feed from Autofocus",
```
Comment (bakatzir) [inline Packs/FeedAutofocus/pack_metadata.json:3]: please add the desc from the integration yml to here as well.

**Comment by kirbles19** [comment]
@orhovy Done.

**Pull request merged**